### PR TITLE
Update workspace management to add impacted stakeholders

### PR DESCRIPTION
**Pull request by klaraavo**
Also removing the "and event variants" here – this is more high level and I don't think we need to differentiate between the two or require the user to parse that concept on this page

### CODE REVIEW

**Comment by klaraavo** [comment]
@logason I'm rejecting your changes to add a bunch of "and variants" to the copy here. Fundamentally, a variant IS an event, just a different shape of it. This article is more high-level and I don't want to complicate things with Avo features or functionalities that people might not get yet. It also just makes the copy more clunky. WDYT

**Comment by bragur** [comment]
I have to admit I find it slightly exclusive not mentioning variants, because event though, as you say, they are fundamentally just another event, at the moment we don’t recognize them as such in the UI, i.e. in the Events sheet. For now, I think it would be more useful to keep it in the copy but we could make a slight adjustment and say _events and their variants_ instead of _events and variants_. I think that encompasses the meaning of what a variant is at the same time as including them.

**Comment by logason** [comment]
> I have to admit I find it slightly exclusive not mentioning variants, because event though, as you say, they are fundamentally just another event, at the moment we don’t recognize them as such in the UI, i.e. in the Events sheet. For now, I think it would be more useful to keep it in the copy but we could make a slight adjustment and say _events and their variants_ instead of _events and variants_. I think that encompasses the meaning of what a variant is at the same time as including them.

+1, at least I think it's very important we make it clear this functionality is available on variants somehow

**Comment by klaraavo** [comment]
My take is that this is mentioned everywhere on the actual page for domain stakeholders. So it is covered in more detail where relevant but I don't think we need to mention it in every single place as it just makes the text harder to read. Making a suggestion with a compromise: 
- We add it in at the top of the article, saying Events (including event variants) and link to event variants in the text
- We add it in the section for attaching events to domains

File: pages/workspace-management/domains.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import PageLink from '../../components/PageLink';
 
 # Managing Domains
 
-Domains give you a clear overview of areas of responsibility and ownership within your schema, by mapping out which domains are stakeholders of each event and event variant in your tracking plan and which users belong to those domains.
+Domains give you a clear overview of areas of responsibility and ownership within your schema, by mapping out which domains are stakeholders of each event in your tracking plan and which users belong to those domains.
```
Comment (klaraavo) [inline pages/workspace-management/domains.mdx:5]:
```suggestion
Domains give you a clear overview of areas of responsibility and ownership within your schema, by mapping out which domains are stakeholders of each event (including [event variants](/data-design/avo-tracking-plan/event-variants)) in your tracking plan and which users belong to those domains.
```

**Review by logason — APPROVED** [review]
LGTM!

**Pull request merged**